Documentation/configuration/sctp.rst

```diff
@@ -0,0 +1,28 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _sctp:
+
+*******************
+SCTP support (BETA)
+*******************
+
+Enabling
+===================
+Pass ``--set sctp.enabled=true`` to helm.
+
+Limitations
+===================
+Cilium supports basic SCTP support. Specifically, the following is supported:
+ - Pod <-> Pod communication
+ - Pod <-> Service communication [*]
+
+[*] SCTP support does not support rewriting ports for SCTP packets. This means
+that when defining services, the targetPort **MUST** equal the port, otherwise
+the packet will be dropped.
```

Comment:
In @pchaigno's comment [here](https://github.com/cilium/cilium/pull/20033/files#r895211322), there is a longer list of current limitations of SCTP support with various Cilium features. Does it make sense to list each of those here to make it clear what kind of incompatibilities exist?